config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-kind-periodics.yaml

```diff
@@ -44,17 +44,11 @@ periodics:
       - name: KIND_CLUSTER_LOG_LEVEL
         # Default is 4, but we want to exercise more log calls and get more output.
         value: "6"
-      - name: FEATURE_GATES
-        value: '{"DynamicResourceAllocation":true,"ContextualLogging":true}'
```

Comment:
is `ContextualLogging` disabled on purpose too?

Reply:
It's now on by default for a few releases, so the job no longer needs to set it explicitly.